testsuite/test_config_vpn.py

```diff
@@ -0,0 +1,137 @@
+import pytest
+from rest_framework.test import APIClient
+from django.contrib.auth.models import User
+from liquidcore.config.models import VPNClientKey
+
+pytestmark = [pytest.mark.django_db, pytest.mark.usefixtures('mock_agent')]
+
+@pytest.fixture
+def client():
+    USERNAME = 'admin'
+    PASSWORD = 'q' * 20
+    User.objects.create_user(
+        username=USERNAME,
+        password=PASSWORD,
+        is_staff=True,
+    )
+    api_client = APIClient()
+    assert api_client.login(username=USERNAME, password=PASSWORD)
+    return api_client
+
+OVPN_CONTENT_TYPE = 'application/x-openvpn-profile'
+
+def test_vpn_system_status(client):
+    vpn_status = client.get('/api/vpn/')
+    assert 200 == vpn_status.status_code
```

Comment:
Mindfuck. It looks like you're checking that the number 200 is correct, and `vpn_status.status_code` is your reference value.